zone/aggro.cpp

```diff
@@ -1278,6 +1295,39 @@ bool Mob::CheckLosFN(glm::vec3 posWatcher, float sizeWatcher, glm::vec3 posTarge
 	return zone->zonemap->CheckLoS(posWatcher, posTarget);
 }
 
+bool Mob::CheckPositioningLosFN(Mob* other, float posX, float posY, float posZ) {
```

Comment:
Just use `x` `y` `z` for params or pass in `glm::vec3` type which would hold all 3